common/src/backends/client/resource_recorder.rs

```diff
@@ -41,6 +42,7 @@ impl<T> ResourceDal for &mut T
 where
     T: ResourceDal,
 {
+    #[instrument(skip_all, fields(shuttle.project_id = project_id))]
```

Comment:
We haven't established a convention for this field yet, although we do call it `shuttle.project_id` in the deployer container labels. Should we perhaps go for `shuttle.project.id` instead?

Reply:
Ah, yes sorry missed that :+1: